### ISSUE TITLE
Cannot edit Calendar or events

### ISSUE THREAD

**Issue by MonsterITServices** [waran70/YetiForce#2]
Hi, I cant seem to edit and add items to the calendar or events module. Is there a way of enabling this please?

**Comment by dextras** [comment]
Hi! As in the previous ticket - add gif for better visualization. Also try the gitdeveloper.yetiforce.com if the problem also occurs there.

**Comment by MonsterITServices** [comment]
![calendar issue](https://cloud.githubusercontent.com/assets/22075161/18438422/38934f30-78f9-11e6-89be-607fa16e2a1d.gif)

**Comment by dextras** [comment]
I believe they are locked for a reason. Probably for certain functionality to work properly you should not be able to modify existing fields. I do not understand why the addition of new fields is not supported though. You know ... editing database always works ;)

**Comment by MonsterITServices** [comment]
Wish I knew how to edit the database to enable or remove fields

**Comment by dextras** [comment]
Lets wait for the developers to comment on this issue.

**Comment by MonsterITServices** [comment]
Just going through the database now to try and find the lockdown but it's been 15 years since i fully edited a database

**Comment by dextras** [comment]
I am using phpmyadmin for database browsing. There are two tables you have to look at: the vtiger_field table and the vtiger_activity. The first shows you all the fields for all the modules and the latter contains all the activities you have already entered. The vtiger_field has to be edited to achieve what you want. But remember that messing with database could seriously break the whole installation. Better do a backup before.

you can issue the following sql command to get the activity fields: SELECT \* FROM `vtiger_field` where tablename="vtiger_activity" ORDER BY `fieldlabel` ASC

**Comment by MonsterITServices** [comment]
Brilliant will have a look at it. Just going through each field that may be related but will check that out thank you. Shame the developer is not around at the moment to help us figure it out.

**Comment by dextras** [comment]
I do not think it is a shame - They are on Holiday. You need a break from work from time to time :)

**Comment by MonsterITServices** [comment]
Ah fair do's didn't know they were on a holiday fair play and hope they enjoy the holiday.

**Comment by dextras** [comment]
#3519

**Comment by MonsterITServices** [comment]
:) thank you. Well I'll dig around and if I find how todo the edit events I'll post up and let you know.
